stone_cx/simulatorRobotDeadReckon.cc

```diff
@@ -55,8 +59,23 @@ void imuThreadFunc(std::atomic<bool> &shouldQuit, std::atomic<float> &heading, u
 
 int main(int argc, char *argv[])
 {
+    // Simulation rendering parameters
+    const unsigned int activityImageWidth = 500;
+    const unsigned int activityImageHeight = 1000;
     const float velocityScale = 1.0f / 10.0f;
+
+    const bool streamActivity = (argc > 1);
+
+    // Create server and sink for sending activity image over network
+    Net::Server server(Net::Socket::DefaultListenPort);
```

Comment:
You don't need to specify the default port here, but it obviously doesn't matter.